extension/app/ts/utils/ethereum.ts

```diff
@@ -50,15 +50,25 @@ export interface IUnsignedTransaction1559 {
 	}[]
 }
 
-export interface ITransactionSignature {
+export interface ITransactionSignatureLegacy {
+	readonly r: bigint
+	readonly s: bigint
+	readonly v: bigint
+	readonly hash: bigint
```

Comment:
Is this the hash of the unsigned transaction or the signed transaction?

Reply:
it should be signed transactions hash

Reply:
maybe worth specifying, but I don't unsigned hashes are used anywhere except in the signing process?